frontend/src/lib/lemon-ui/Tooltip/Tooltip.tsx

```diff
@@ -32,6 +32,7 @@ export interface TooltipProps {
     placement?: Placement
     className?: string
     visible?: boolean
+    interactive?: boolean
```

Comment:
Can you talk me through what this change does? E.g. including a video of before/after in the PR?

Reply:
This allows us to hover over a tooltip, and interact with the content inside (copy the text, click on a link, toggle, button, etc.). I'm copying the term `interactive` from MaterialUI, I'm open to making it more clear to our own terms if you think the name is misleading.

I'll add a video to the PR description.